internal/version/version.go

```diff
@@ -12,40 +12,62 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package version declares types for each string to keep strict coupling to the JSON schema
```

Comment:
this is a comment which can be used to squarely blame adrian for the type aliases later. If they become a problem I won't complain much about reverting them.. I also usually find type aliases annoying